packages/sdk-codegen/src/kotlin.gen.ts

```diff
@@ -58,6 +58,8 @@ export class KotlinGen extends CodeGen {
   needsRequestTypes = false
   willItStream = true
 
+  tFailure = 'com.looker.sdk.Error'
```

Comment:
I used the fully qualified name here, since Kotlin also defines an `Error` class.  Maybe the API response type should be clarified to use something that doesn't conflict.

Reply:
We rename the spec's `Error` class for Swift, which is less flexible. We may need to do the same kind of thing for the Kotlin generator.